.github/workflows/slash-command-dispatch.yml

```diff
@@ -0,0 +1,19 @@
+name: Slash Command Dispatch
+on:
+  issue_comment:
+    types: [created]
+jobs:
+  slashCommandDispatch:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Slash Command Dispatch
+        uses: peter-evans/slash-command-dispatch@v2
+        with:
+          issue-type: pull-request
+          permission: write
+          # This token uses the public_repo scope to create repository_dispatch and workflow_dispatch events.
+          # This particular personal access token is tied to this account: https://github.com/aprea
+          # Note: GITHUB_TOKEN does not work here.
+          token: ${{ secrets.SLASH_COMMAND_ACCESS_TOKEN }}
+          commands: |
+            build
```

Comment:
How does this command link to `Slash Command Build`? I can see from the yml that the "input" is `/build` in a comment on a PR, but it's not clear to me how that triggers the next step (i.e. the output of the action).

Might be worth adding a docblock of explicit inputs, outputs and expected behaviour, and/or linking to docs for `peter-evans/slash-command-dispatch@v2`.